packages/serve/src/createConfig.ts

```diff
@@ -10,6 +10,15 @@ import { devServerOptionsType } from './types';
  */
 export default function createConfig(args): devServerOptionsType {
     const options = { ...args };
+    let isDevServer4 = false,
```

Comment:
Let's keep `webpack-dev-server` check in a single place.
It is in `index.ts` as well as this file.

Reply:
@evilebottnawi suggested to keep it here too

Reply:
When availability of `webpack-dev-server` is already tested in `index.ts` which is entry file to `serve` package, why do we need to test it again?

Reply:
https://github.com/webpack/webpack-cli/pull/2022#discussion_r516001135

Reply:
Okay, lets keep it then.